Makefile

```diff
@@ -0,0 +1,30 @@
+
+DOCKER ?= $(shell which docker)
+DOC_FILES := \
+	README.md \
+	code-of-conduct.md \
+	ROADMAP.md \
+	implementations.md \
+	bundle.md \
+	runtime.md \
+	runtime-linux.md \
+	config.md \
+	config-linux.md \
+	runtime-config.md \
+	runtime-config-linux.md
```

Comment:
On Wed, Dec 09, 2015 at 09:37:07AM -0800, Tianon Gravi wrote:
> > +DOC_FILES := \
> > +	README.md \
> > +	code-of-conduct.md \
> > +	ROADMAP.md \
> > +	…
> > +	runtime-config-linux.md
> 
> Is maintaining this hard-coded list going to be a hassle?  (ie,
> would something like `$(wildcard *.md)` also be correct without the
> same maintainer cost?)

It looks like the ordering is significant, so the wildcard approach
would require numbering the filenames (e.g. 01-README.md).  I think
the hard-coded approach is fine, since filename churn has been small
so far, and will only get smaller as the repository matures (excepting
a few isolated reorgs).